.github/workflows/build_pages.yml

```diff
@@ -0,0 +1,75 @@
+name: deploy-book
+
+# Run this when the master or main branch changes
+on:
+  push:
+    branches:
+    - main
+    # If your git repository has the Jupyter Book within some-subfolder next to
+    # unrelated files, you can make this run only if a file within that specific
+    # folder has been modified.
+    #
+    # paths:
+    # - some-subfolder/**
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
You can manually rebuild by going to Actions tab, selecting the `deploy-book` action and clicking `Run workflow`